### ISSUE TITLE
CSS minify problem with calc()

### ISSUE THREAD

**Issue by TriForceX** [TriForceX/WebsiteBase#14]
Theres a problem with the PHP minify code for CSS when tries to minify calc(). The code is is only keep the spaces in * and / symbols but not in + and - so it will not work.

This code:
```css
.my-style {
   background-position: calc(100% - 50px) calc(100% / 2);
   height: calc(20.5rem + 100%);
}
```
Will result in :
```css
.my-style {
   background-position: calc(100%-50px) calc(100% / 2);
   height: calc(20.5rem+100%);
}
```

The operator symbols without spaces between them will not work.

**Comment by TriForceX** [comment]
Im working on small workarround to check this, i added the next regexp:
```
calc\((.*?)([\+\-\*])(.*?)\)
```
And replacing with:
```
calc($1 $2 $3)$4
```

The code now works fine and now is adding spaces between + and - but if you have more than 1 operator inside a calc() it will work only once:
```
.my-style {
   height: calc(20.5rem + 100% + 30px - 2px - 1px);
}
```
Will result in:
```
.my-style {
   height: calc(20.5rem + 100%+30px-2px-1px);
}
```

**Comment by TriForceX** [comment]
Problem solved in the main regexp (commit c5b7b67). This workarround was removed.
